src/Hl7.Fhir.Base/Model/InterfacesAppliedToGeneratedClasses/IIdentifiable.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Hl7.Fhir.Generation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Hl7.Fhir.Model
+{
+    [ApplyInterfaceToGeneratedClasses]
+    public interface IIdentifiable
+    {
+        List<Identifier> Identifier { get; set; }
+    }
+
+    public static class IdentifiableExtensions
```

Comment:
not sure if this should be kept, I wanted to make an example with extension methods.